src/components/BrowseSkill/custom.css

```diff
@@ -23,9 +23,25 @@
 }
 
 div.scrolling-wrapper::-webkit-scrollbar {
-        display: none;
+    display: none;
 }
 
 .clear-button:hover {
     color: #c45500;
 }
+
+.index-link-sidebar {
+    font-size: 15px;
+    width: fit-content;
+    color: black;   
+}
+
+.index-link-sidebar:hover {
```

Comment:
Please change this hover cover according to theme.

Reply:
Which color should be used? I just took alexa's hover color.